jobs/SDE/Assignments/SDE-FE-Assignment.md

```diff
@@ -0,0 +1,76 @@
+### SDE coding challenge
+
+Here you are! We're happy that you're interested in working for Quantum Inventions! 
+To get a better idea of your current skills, we'd like you to complete a coding challenge - build a web app to display and update resources (e.g - users). 
+The actual requirements are listed further down, but here are the general tech requirements:
+- Make sure your app is runnable on Windows, Mac OS X and Linux
+- Do not include any paid software dependencies
+- Include instructions for setting up and running your application
+- Containerized app is a plus
+
+### Mock APIs 
+
+Simulate API like behaviour using following approach -
+* You can use API mock service: https://reqres.in/. Visit the site for more details.
+
+### Description 
```

Comment:
I would like to understand the list of skills/practices/design/expertise/etc. you are looking to obtain from the below assignment. Could you list them here?

Reply:
Listing down the expectations, we can do in `### What we care about` section. cause if I am correct the description, is about describing the problem to solve.

Reply:
I actually meant for you to list the points as a comment here for me to understand but your suggestion would be best

Reply:
Since the **What we're looking for** section is open-ended (as intended), can you comment here on what kind of answers to those points you would approve? This is for my understanding of how you are testing the candidate.